src/Tests/Tests.fs

```diff
@@ -123,3 +123,35 @@ let ``Should generate valid Uatp`` () =
     card |> should startWith "1"
     card |> should haveLength 15
     card |> luhn |> should be LuhnCheck
+
+
+[<Theory>]
+[<InlineData(DinersClubInternationalLengthOptions.Fourteen, 14)>]
+[<InlineData(DinersClubInternationalLengthOptions.Fifteen, 15)>]
+[<InlineData(DinersClubInternationalLengthOptions.Sixteen, 16)>]
+[<InlineData(DinersClubInternationalLengthOptions.Seventeen, 17)>]
+[<InlineData(DinersClubInternationalLengthOptions.Eightteen, 18)>]
+[<InlineData(DinersClubInternationalLengthOptions.Nineteen, 19)>]
+let ``Should generate valid DinersClubInternational`` length expectedLength =
+    let card = Cardizer.NextDinersClubInternational length
+    let start = card.Substring(0, 2) |> int
+    let prefixInRange = start = 36
+    prefixInRange |> should be True
```

Comment:
Like you did for `NextDinersClubUsAndCanada`, use `card |> should startWith "36"` instead.